ios-training-bjung/HomePresenter.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  HomePresenter.swift
+//  ios-training-bjung
+//
+//  Created by 鄭 普勤 on 2023/12/01.
+//
+
+import Foundation
+import YumemiWeather
+import UIKit
+
+protocol HomePresenterInput {
+    @MainActor
+    func loadWeatherData()
+}
+
+protocol HomePresenterOutput: AnyObject {
+    func updateInfoDisplay(imageResId: String, color: UIColor)
+}
+
+final class HomePresenter: HomePresenterInput {
+    private weak var view: HomePresenterOutput!
```

Comment:
injectはcompile保証できないため
```suggestion
    private weak var view: HomePresenterOutput?
```

Reply:
@novr 
2d314577b5504cfae092d45455392b1b59fa6221
上記のコミットで対応しました。